pkg/vcs/vcs.go

```diff
@@ -64,6 +64,10 @@ type Repo interface {
 
 // Bisecter may be optionally implemented by Repo.
 type Bisecter interface {
+	// Can be used for last minute preparations like pulling release tags into the bisected repo, which
+	// is required to determin the compiler version to use on linux. Can be an empty function.
+	BisectPreparation() error
```

Comment:
Nit: maybe `PrepareBisect` stays more in line with the other methods?

Reply:
Ack, renamed